app/views/spendings/index.html.erb

```diff
@@ -0,0 +1,42 @@
+<section class='mainCover'>
+    <section class='mainCoverTwo'>
+    <section class='header'>
+    <div class='headerOne'>
+      <div><%=link_to categories_path do %> <%= image_tag("arrow.png", :alt => "menu feed", class: "imageElement") %> <%end%> </div>
+      <div>Transactions</div>
+      <div>Add</div>
+    </div> 
+  </section>
+          <div class="mainContainer">
+<div class="">
+  <% if @spendigs.empty?%>
+    <p class="">You don't have any transactions yet !!</p>
+        <% else %>
+            <ul class="">
+                <li class="detailsTitle">
+                    <div class="detailsCategoryIcon">
+                        <div class=" "><%= @category.icon %></div>
+                        <div class=" "><%= @category.name %></div>
+                    </div>
+                    <div class=" ">
+                         <div class="">Total spent on this category</div>
+                        <div class="">$ <%= number_with_delimiter(@total) %></div>
+                    </div> 
+                </li>
+                <% @spendigs.each do |ex| %>
+                    <li class="detailsList">
+                        <div class=" ">
+                            <span class=" "><%= ex.name %></span>
+                            <span class=" ">$ <%=  number_with_delimiter(ex.amount)  %></span>
+                        </div>
+                        <div class=" ">
+                            <span class=" "><%= ex.created_at.strftime("%d %B '%Y, %R") %></span>
+                        </div>
+                    </li>
+                <% end %>
+            </ul>
+  <% end %>
+  <div class='categoryButton userSubmitContainer'><%= link_to "Add a new Transaction", new_category_spending_path(@category), class: '' %></div>
+</div>
```

Comment:
- [ ] It would be nice if you could place the button at the bottom of the page just as it is in the original design.

<img width="449" alt="Screenshot 2023-10-21 at 4 47 57 PM" src="https://github.com/tonnytech/Budget_App/assets/50941074/2ec068c4-5b90-4688-9be3-e066ca74647b">